app/frontend/components/pages/accounts/accountsDashboard.tsx

```diff
@@ -0,0 +1,296 @@
+import {Fragment, h} from 'preact'
+import {connect} from '../../../helpers/connect'
+import actions from '../../../actions'
+import range from '../../../../frontend/wallet/helpers/range'
+import printAda from '../../../helpers/printAda'
+import {Lovelace, State} from '../../../state'
+import {AdaIcon} from '../../common/svg'
+import Alert from '../../common/alert'
+import SendTransactionModal from './sendTransactionModal'
+import DelegationModal from './delegationModal'
+import ConfirmTransactionDialog from '../../../../frontend/components/pages/sendAda/confirmTransactionDialog'
+import {errorHasHelp} from '../../../../frontend/helpers/errorsWithHelp'
+import TransactionErrorModal from '../sendAda/transactionErrorModal'
+import {getTranslation} from '../../../../frontend/translations'
+
+const AccountTile = ({
+  accountIndex,
+  account,
+  setSelectedAccount,
+  selectedAccountIndex,
+  showDelegationModal,
+  showSendTransactionModal,
+  setLastAccountExplored,
+  isLastAccountExplored,
+}) => {
+  const isSelected = selectedAccountIndex === accountIndex
+
+  const shouldShowAccountInfo =
+    account && (account.isUsed || isLastAccountExplored || accountIndex === 0)
```

Comment:
under which conditions is the account undefined? Is it when bulk key export is not allowed?